web/src/hashchange.ts

```diff
@@ -237,6 +238,26 @@ function do_hashchange_normal(from_reload: boolean, restore_selected_id: boolean
         case "#feed":
             show_all_message_view();
             break;
+        case "#topics":
+            // Show inbox style topics list for the channel if
+            // the channel id is valid.
+            if (hash.length === 3) {
+                const channel_id_string = hash[2];
```

Comment:
I think we should plan to use/support the same `{id}-{name}` syntax we use in narrow URLs. Can be fine to add that as a follow-up, but FYI in case it's easy to do.

Reply:
Done.